lists_hw/MyLinkedList.java

```diff
@@ -0,0 +1,110 @@
+package ListLab;
+
+import java.util.Iterator;
+
+public class MyLinkedList<T> implements Iterable<T>{
+
+	private Node<T> head;
+	private int size = 0;
+
+	public int getSize() {
+		return size;
+	}
+
+	public boolean remove(int index) {
+		//to-do
+		if (index > size) {
+			return false;
+		}
+		Node<T>node = head;
+		Node<T>afterNode;
+		for (int i = 0; i < index-1; i++) {
+			node = node.getNext();
+		}
+		afterNode = node.getNext();
+		node.setNext(afterNode);
```

Comment:
I think this is the equivalent of `node.setNext(node.getNext());`, which wouldn't change anything. Did you mean `node.setNext(afterNode.getNext());`? Also, will this handle the case when the user tries to remove the head node at index 0?